pyanaconda/installclasses/rhv.py

```diff
@@ -0,0 +1,132 @@
+#
+# rhv.py
+#
+# Copyright (C) 2016  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from blivet.devicefactory import DEVICE_TYPE_LVM_THINP
+from blivet.size import Size
+from pykickstart.constants import AUTOPART_TYPE_LVM_THINP
+
+from pyanaconda import network
+from pyanaconda import nm
+from pyanaconda.core.constants import STORAGE_ROOT_DEVICE_TYPES, STORAGE_MUST_NOT_BE_ON_ROOT, \
+    STORAGE_REQ_PARTITION_SIZES
+from pyanaconda.installclass import BaseInstallClass
+from pyanaconda.kickstart import getAvailableDiskSpace
+from pyanaconda.platform import platform
+from pyanaconda.product import productName
+from pyanaconda.storage.autopart import swap_suggestion
+from pyanaconda.storage.partspec import PartSpec
+
+__all__ = ["OvirtBaseInstallClass", "RHEVInstallClass"]
+
+
+class OvirtBaseInstallClass(BaseInstallClass):
+    name = "oVirt Node Next"
+    sortPriority = 21000
+    hidden = not productName.startswith("oVirt")
+
+    efi_dir = "centos"
+    default_autopart_type = AUTOPART_TYPE_LVM_THINP
+
+    # there is a RHV branded help content variant
+    help_folder = "/usr/share/anaconda/help/rhv"
+
+    def configure(self, anaconda):
+        BaseInstallClass.configure(self, anaconda)
```

Comment:
Is this really required. Wouldn't it be called automatically if not specified?

Reply:
Removed.